content/en/docs/spinnaker/terraform-enable-integration.md

```diff
@@ -256,14 +256,59 @@ spec:
       armory:
         terraform:
           enabled: true
+          # Remote backends is an Early Access feature. Do not use it in production. Omit this field if you do not want to use remote backends.
+          remoteBackendSupport: true
```

Comment:
We don't have support for this in Armory Halyard, yet. It would need to be added to `terraformer-local.yml`.

Reply:
This section is for Operator. Do you mean Operator can't enable it yet?

Reply:
I think that Operator doesn't know how to enable this yet either.